lib/modules/manager/npm/extract/common/dependency.ts

```diff
@@ -98,14 +97,15 @@ export function extractDependency(
   if (dep.currentValue.startsWith('npm:')) {
     dep.npmPackageAlias = true;
     const valSplit = dep.currentValue.replace('npm:', '').split('@');
-    if (valSplit.length === 2) {
+    if (valSplit.length === 1) {
+      dep.packageName = depName;
+      dep.currentValue = valSplit[0];
+    } else if (valSplit.length === 2) {
       dep.packageName = valSplit[0];
       dep.currentValue = valSplit[1];
     } else if (valSplit.length === 3) {
       dep.packageName = valSplit[0] + '@' + valSplit[1];
       dep.currentValue = valSplit[2];
-    } else {
-      logger.debug('Invalid npm package alias: ' + dep.currentValue);
```

Comment:
What happens now if the split is >3? Silently discarded?

Reply:
Is there a real world example of such a case where `currentValue` with `npm:` stripped and split with `@` can be >3?

Reply:
The point is that it's an error case

Reply:
Sure, can add that back :)